### PR TITLE
chore: improve wordings of capability question model

### PR DESCRIPTION
**Pull request by a1exwang**
- improve wordings for vscode extension (all changes are under `isBotNotificationEnabled` feature flag
- move feature flags to a new file to prevent circular dependency (re-export in the original position to keep compatible)

Create new:
image

Add capability:
image

Improve wordings according to: [this one note](https://microsoftapc.sharepoint.com/teams/DevDivTeamsDevXProductTeam/_layouts/OneNote.aspx?id=%2Fteams%2FDevDivTeamsDevXProductTeam%2FShared%20Documents%2FService%20infrastructure%20Team%2FInfra%20team&wd=target%28Infra.one%7C54126BB5-B660-493E-ABC2-5CDF1390B13A%2FUX%7C45A09606-1E4A-43D5-8624-D8E137A2833B%2F%29)

### CODE REVIEW

**Comment by a1exwang** [comment]
E2E TEST: n/a

**Review by Siglud — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/question.ts [hunk 1]
```diff
@@ -9,14 +9,19 @@ import {
...
   Void,
 } from "@microsoft/teamsfx-api";
+import { isBotNotificationEnabled } from "../../../common/featureFlags";
 import { getLocalizedString } from "../../../common/localizeUtils";
 
 export const TabOptionItem: OptionItem = {
   id: "Tab",
-  label: "Tab",
+  label: isBotNotificationEnabled() ? "SSO enabled Launch page" : "Tab",
   cliName: "tab",
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/plugins/solution/fx-solution/question.ts:18]: Does this apply to cli? Or the wording between vscode and cli will be different.

Comment (a1exwang) [reply packages/fx-core/src/plugins/solution/fx-solution/question.ts:18]: This PR only changes VSC, CLI UX is not finalized yet.

**Review by KennethBWSong — DISMISSED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/question.ts [hunk 2]
```diff
@@ -29,42 +34,47 @@ export const BotOptionItem: OptionItem = {
...
+  label: getLocalizedString("core.CommandAndResponseOption.label"),
   cliName: "command-and-response",
-  description: getLocalizedString("core.CommandAndResponseOption.description"),
   detail: getLocalizedString("core.CommandAndResponseOption.detail"),
 };
 
 export const ExistingTabOptionItem: OptionItem = {
   id: "ExistingTab",
-  label: "Existing Tab",
+  label: "Embed existing web app",
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/plugins/solution/fx-solution/question.ts:51]: Move to package.nls.json for localization?

**Review by nliu-ms — DISMISSED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/question.ts (hunk 2 above)
Comment (a1exwang) [reply packages/fx-core/src/plugins/solution/fx-solution/question.ts]: updated and tested

**Review by nliu-ms — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by Siglud — APPROVED** [review]

**Pull request merged**